README.md

```diff
@@ -64,40 +65,115 @@ Flags:
 ```
 
 Every argument can be defined using environment variables using `ROSETTA_` prefix; and replacing `.` for `_`; for example:
-```
+
+```sh
 ROSETTA_DATADIR="/my/dir"
 ROSETTA_GETH_GENESIS="/path/to/genesis.json"
-``` 
+```
+
+## Running the Rosetta RPC Server
+
+Running the Rosetta RPC Server from scratch will take some time to sync, since it runs a full archive node in the background. While it may be possible to run the Construction API in the future with a non-archive node, this is still required by the Rosetta spec for the Data API implementation in order to perform balance reconciliation.
+
+### Version 1: Running from `rosetta` source code
+
+You will need the following three repositories cloned locally:
+
+* `rosetta` (this repo)
+* [`celo-monorepo`](https://github.com/celo-org/celo-monorepo)
+* [`celo-blockchain`](https://github.com/celo-org/celo-blockchain)
+
+You also need the following dependencies to be met:
+
+* `go >= 1.14`
+* `rust >= 1.41.0` (`blockchain` dependency)
+* `node = 10` (`celo-monorepo` dependency)
+* `golangci` ([installation instructions](https://golangci-lint.run/usage/install/#local-installation)) (linter dependency for the Makefile)
+
+#### Running on Alfajores (Testnet)
+
+Prerequisites:
+
+* Checkout `celo-monorepo` branch `alfajores` and run `yarn && yarn build --ignore docs`
+* Checkout `celo-blockchain` tag `v1.1.2` (`git fetch --all && git checkout v1.1.2`) (NOTE: check that this matches the version specified in the `rosetta` `go.mod` file) and `make all`
+* Set paths to `celo-monorepo` and `celo-blockchain` as `CELO_MONOREPO_PATH` and `CELO_BLOCKCHAIN_PATH` respectively (paths can be absolute or relative to the `rosetta` repo. If desired, add these lines to your bash profile:
+
+  ```sh
+  export CELO_MONOREPO_PATH=path/to/celo-monorepo
+  export CELO_BLOCKCHAIN_PATH=path/to/celo-blockchain
+  ```
+
+* Checkout `rosetta` tag `v0.7.6` (`git fetch --all && git checkout v0.7.6`) (or latest released tag) and `make gen-contracts && make all`
+* Run `make alfajores-env` to create an empty datadir with the genesis block (only needs to be run the first time, upon initializing the service)
+
+Then run:
+
+```bash
+go run main.go run \
+  --geth.genesis ./envs/alfajores/genesis.json \
+  --geth.binary ../celo-blockchain/build/bin/geth \
+  --geth.syncmode full \
+  --geth.gcmode archive \
+  --geth.staticnodes "enode://05977f6b7d3e16a99d27b714f8a029a006e41ec7732167d373dd920d31f72b3a1776650798d8763560854369d36867e9564dad13b4b60a90c347feeb491d83a9@34.83.42.50:30303" \
+  --datadir "./envs/alfajores"
+```
 
```

Comment:
Add:

Your system will prompt you to Deny or Allow the application to accept incoming network connections.

You should start to see continuous output similar to this:

```sh
INFO [01-28|14:09:03.434] Press CTRL-C to stop the process
INFO [01-28|14:09:03.440] Running geth init                        service=geth
INFO [01-28|14:09:04.104] Geth Already initialized... skipping init service=geth
geth --networkid 44787 --nousb --rpc --rpcaddr 127.0.0.1 --rpcport 8545 --rpcvhosts localhost --syncmode full --gcmode archive --rpcapi eth,net,web3,debug,admin,personal --ipcpath <YourPathToRosetta>/rosetta/envs/alfajores/celo/geth.ipc --light.serve 0 --light.maxpeers 0 --maxpeers 1100 --consoleformat term
INFO [01-28|14:09:05.110] Detected Chain Parameters                chainId=44787 epochSize=17280
INFO [01-28|14:09:05.120] Starting httpServer                      listen_address=:8080
INFO [01-28|14:09:05.120] Resuming operation from last persisted  block srv=celo-monitor block=0
INFO [01-28|14:09:05.121] SubscriptionFetchMode:Start              srv=celo-monitor pipe=header_listener start=1

...

INFO [01-28|14:09:25.731] Stored 1000 blocks                       srv=celo-monitor pipe=persister       block=1000 registryUpdates=0
INFO [01-28|14:09:34.101] Stored 1000 blocks                       srv=celo-monitor pipe=persister       block=2000 registryUpdates=0
INFO [01-28|14:09:42.625] Stored 1000 blocks                       srv=celo-monitor pipe=persister       block=3000 registryUpdates=0
INFO [01-28|14:09:50.936] Stored 1000 blocks                       srv=celo-monitor pipe=persister       block=4000 registryUpdates=0
INFO [01-28|14:09:59.188] Stored 1000 blocks                       srv=celo-monitor pipe=persister       block=5000 registryUpdates=0
INFO [01-28|14:10:07.479] Stored 1000 blocks                       srv=celo-monitor pipe=persister       block=6000 registryUpdates=0

...

```

Reply:
✅